common/src/main/resources/db/changelog/test_only/coverage-fake-partitions.sql

```diff
@@ -6,7 +6,7 @@
 -- May be removed or altered in future once cutover to partitioned coverage table is complete
 -- this is used for testing code changes locally
 
-create table if not exists tst partition of coverage_partitioned
+create table if not exists tst partition of coverage
```

Comment:
Is changing an sql file going to cause issues with the db?

Reply:
I doubt it because this is only applied in the 'test' context so it is only applied locally when running unit and integration tests.

Reply:
I fixed this after realizing what you meant. Yeah it is an error on my part.